DEVELOPER_GUIDE.md

```diff
@@ -265,6 +265,26 @@ Here, we are testing BWC Tests with BWC version of plugin as 1.0.0.0. Make sure
 
 Before adding any new tests to Backward Compatibility Tests, we should be aware that the tests in BWC are not independent. While creating an index, a test cannot use the same index name if it is already used in other tests. Also, adding extra operations to the existing test may impact other existing tests like graphCount. 
 
+## Codec Versioning
```

Comment:
Can you add this(with a hyperlink) to the index on the top of this file ?

Reply:
ack